articles/mysql/flexible-server/connect-workbench.md

```diff
@@ -30,12 +30,11 @@ This quickstart uses the resources created in either of these guides as a starti
 
 - If you created your flexible server with *Private access (VNet Integration)*, you will need to connect to your server from a resource within the same VNet as your server. You can create a virtual machine and add it to the VNet created with your flexible server. Refer to [Create and manage virtual networks for Azure Database for MySQL - Flexible Server using the Azure CLI](how-to-manage-virtual-network-cli.md).
 - If you created your flexible server with *Public access (allowed IP addresses)*, you can add your local IP address to the list of firewall rules on your server. Refer to [Manage firewall rules for Azure Database for MySQL - Flexible Server using Azure CLI](how-to-manage-firewall-cli.md).
-
 - Download and install MySQL Workbench on your computer from [the MySQL website](https://dev.mysql.com/downloads/workbench/).
 
 ## Get connection information
 
-Get the connection information needed to connect to the flexible server. You need the fully qualified server name and sign in credentials.
+Get the connection information needed to connect to the Flexible Server. You need the fully qualified server name and sign in credentials.
```

Comment:
```suggestion
Get the connection information needed to connect to the flexible server. You need the fully qualified server name and sign in credentials.
```
Using lowercase for a reference that doesn't correspond to a product name.

Reply:
@ShannonLeavitt - Thanks for the suggestion.